### ISSUE TITLE
Using UInt for js target 

### ISSUE THREAD

**Issue by aidangrabe** [pixijs/pixi-haxe#6]
I get the following error when trying to compile:

```
/usr/share/haxe/lib/pixijs/2,1,3/pixi/primitives/Graphics.hx:53: characters 1-73 : Class not found : UInt
```

https://github.com/adireddy/haxe-pixi/blob/master/pixi/primitives/Graphics.hx#L53

I know the documentation says to simulate UInt for platforms that don't support it, but would it maybe be better to change the signature of methods to use Int instead seeing as JS is the only target?

**Comment by adireddy** [comment]
Makes sense. Will change that.

Out of interest, what is your target?

**Comment by adireddy** [comment]
No pixi.js is only for JavaScript. OpenFL is the alternative if you want to target multiple platforms.

**Comment by adireddy** [comment]
Updated and released v2.1.5
